app/page-partials/my-dashboard/UserRoles.tsx

```diff
@@ -216,7 +290,32 @@ const UserRoles = ({ selectedRequest, alert }: Props) => {
           <td>{row.firstName}</td>
           <td>{row.lastName}</td>
           <td>{row.email}</td>
-          <td>{row.username}</td>
+          <td>
+            <td>
```

Comment:
Is there supposed to be nested < td >'s here?

Reply:
right, removed the unnecessary `td`